WasteWar/Assets/Scripts/Drawing/StructurePlacement.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using Constants;
+
+public class StructurePlacement : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject buildingPrefab1;
```

Comment:
this is temporary, we will later use ScriptableObjects - its a big topic so I would suggest start crawling in there